README.md

```diff
@@ -233,6 +233,23 @@ This currently works automatically for the `target/kubernetes/openshift.yml` fil
 
 Note that it is usually a good idea to set `-Dts.image-overrides` to a _full_ path, because Maven changes the current working directory when running tests.
 
+### Native image
+
+The test suite contains a Maven profile configuring required system properties for native image build. The profile is
+activated by specifying the `native` Quarkus profile (`-Dquarkus.profile=native`). This profile does not specify the 
+container runtime for Quarkus builder image, so by default, native build will use local GraalVM. The following command 
+that will execute the whole test suite using Docker to run containers for native build:
+
+```
+./mvnw clean verify \
+  -Dquarkus.profile=native \ 
+  -Dquarkus.native.container-runtime=docker \
+  -Dts.authenticated-registry
+```
+
+Currently used builder image is `quarkus/ubi-quarkus-mandrel` and the base image for OpenShift deployment is 
+`quarkus/ubi-quarkus-native-binary-s2i`.
+
 ### TODO
 
 There's a lot of possible improvements that haven't been implemented yet.
```

Comment:
Add item to avoid usage of `quarkus/ubi-quarkus-native-binary-s2i`, in the end ubi8-minimal should be used for the product

Reply:
`quarkus/ubi-quarkus-native-binary-s2i` is actually `ubi8-minimal` + a few packages + the S2I scripts. I'm not sure if it's meant to stay like this, but if it's a TODO, then primarily somewhere else (and only secondarily in this test suite).

Reply:
It's todo for this TS, ubi-quarkus-native-binary-s2i is for community use, not supported for the product 😎
I'm fine with using it for now because of the reasons you mentioned (being actually ubi8-minimal + few things on top of it). Just want to make sure this item is tracked.